CMakeLists.txt

```diff
@@ -49,7 +56,21 @@ if(WIN32)
 endif()
 find_package(Python3 COMPONENTS Interpreter Development)
 find_package(Qt5 5.15 CONFIG REQUIRED COMPONENTS Core Gui Network Widgets Concurrent)
+
+if(IS_DEBUG_BUILD AND WIN32)
+    # Explicitly setting SHIBOKEN_PYTHON_LIBRARIES variable to avoid PYTHON_DEBUG_LIBRARY-NOTFOUND
+    # link errors on Windows debug builds.
+    set(SHIBOKEN_PYTHON_LIBRARIES ${Python3_LIBRARIES})
+endif()
 find_package(Shiboken2 5.15 CONFIG REQUIRED COMPONENTS libshiboken2 shiboken2)
+
+if(IS_DEBUG_BUILD AND WIN32)
+    # Remove NDEBUG from Shiboken2 INTERFACE_COMPILE_DEFINITIONS so it is not inherited in debug builds.
+    get_property(ShibokenInterfaceDefs TARGET Shiboken2::libshiboken PROPERTY INTERFACE_COMPILE_DEFINITIONS)
```

Comment:
This is needed because /mingw64/lib/cmake/Shiboken2-5.15.13/Shiboken2Targets.cmake has code that unconditionally sets NDEBUG. :(